router/routes/scraper.js

```diff
@@ -11,6 +11,7 @@ var scraper = require('../../modules/scraper_utils');
 
```

Comment:
This is kind of an admin page, all the routes are only explicitly called by me, so you can remove the revert this file